OrbitCore/TracepointEventBuffer.cpp

```diff
@@ -8,7 +8,8 @@ using orbit_client_protos::TracepointEventInfo;
 
 void TracepointEventBuffer::AddTracepointEventAndMapToThreads(uint64_t time,
                                                               uint64_t tracepoint_hash,
-                                                              int32_t thread_id) {
+                                                              int32_t thread_id,
+                                                              bool is_same_pid_as_target) {
   ScopeLock lock(mutex_);
   std::map<uint64_t, orbit_client_protos::TracepointEventInfo>& event_map =
       tracepoint_events_[thread_id];
```

Comment:
so if this tracepoint event is for different process do we want to store it here?

Reply:
Yes. After a discussion with @dpallotti, I was informed that the events should be collected system-wide in the collector because you might hook a tracepoint and since it is not the selected process, it will not be displayed. @dpallotti maybe could give more details about it.

Reply:
When we show events from all processes, we will want a special map for events from all processes, similar to the `event_map_all_threads` that we now have for all threads of the target process. But I don't think that for events from processes different from the target we should group them by thread here (they will all be displayed in one single track). So in other words, I would put this code also in `if (is_same_pid_as_target) {`. Later, when we support events from all threads, we should have the dedicated code outside of such `if`. Does it make sense?

Reply:
Of course! Thank you so much for the suggestion.